### PR TITLE
support FakeXMLHttpRequest response property

### PR DESCRIPTION
**Pull request by kellyselden**
re https://github.com/pretenderjs/FakeXMLHttpRequest/pull/53

These 2 PRs fixed the problem for me https://github.com/ember-vr/ember-vr-shopping/pull/192

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "typescript-eslint-parser": "^21.0.2"
   },
   "dependencies": {
-    "fake-xml-http-request": "^2.0.0",
+    "fake-xml-http-request": "kellyselden/FakeXMLHttpRequest#response",
```
Comment (xg-wang) [inline package.json]: It seems this package used in test is from bower.
https://github.com/pretenderjs/pretender/blob/v3.0.4/karma.conf.js#L18

Comment (xg-wang) [reply package.json]: Just merged https://github.com/pretenderjs/pretender/pull/281. Can you try rebase?

Comment (kellyselden) [reply package.json]: done

**Comment by kellyselden** [comment]
Do you happen to see where I went wrong in the tests or code?

**Comment by xg-wang** [comment]
@kellyselden It seems the changes in `src/fake-xml-http-request.js` should also be made to `fake-xml-http-request.js` under root, that is the file used for testing https://github.com/pretenderjs/pretender/blob/0b0436294f8d07b56e15cabef58adb55359de6ff/karma.conf.js#L18

**Comment by kellyselden** [comment]
@xg-wang Good call. I did that, and it appears the same issue remains.

File: package.json (hunk 1 above)
Comment (xg-wang) [reply package.json]: I think it's github artifact cache didn't update. I just tried v2.1.1 and it passed tests.

**Comment by kellyselden** [comment]
Updated with 2.1.1 and all is good now.

**Pull request merged**